docs/gpa.md

```diff
@@ -233,78 +231,23 @@ The information included in the notice of intended procurement covers all the in
     <tbody>
       <tr class="section">
         <td>X:7</td>
-        <td colspan="2">A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
-      </tr>
-      <tr>
-        <td>X:7(a)</td>
-        <td>the procurement, including the nature and the quantity of the goods or services to be procured or, where the quantity is not known, the estimated quantity and any requirements to be fulfilled, including any technical specifications, conformity assessment certification, plans, drawings or instructional materials;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(b)</td>
-        <td>any conditions for participation of suppliers, including a list of information and documents that suppliers are required to submit in connection with the conditions for participation;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(c)</td>
-        <td>all evaluation criteria the entity will apply in the awarding of the contract, and, except where price is the sole criterion, the relative importance of such criteria;</td>
```

Comment:
We can map tender.awardCriteriaDetails, map tender.awardCriteria (at least in the case of *price is the sole criterion*, and mention the [award criteria breakdown](https://extensions.open-contracting.org/en/extensions/awardCriteria/master/) extension in case they have structured data.

Reply:
The Award Criteria extension only applies to lots, and we have never spawned lots in the GPA guidance. I'm not sure how we would use this extension in GPA, and I think switching to a lot-oriented guidance, like in the EU profile, is overkill for GPA.

Unless we want to push for lot-oriented data structures everywhere.

Reply:
We can at least do the first two suggestions:

> We can map tender.awardCriteriaDetails, map tender.awardCriteria (at least in the case of *price is the sole criterion*

Reply:
Done